lib/marked.js

```diff
@@ -630,7 +627,7 @@ inline.gfm.url = edit(inline.gfm.url, 'i')
 
 inline.breaks = merge({}, inline.gfm, {
   br: edit(inline.br).replace('{2,}', '*').getRegex(),
-  text: edit(inline.gfm.text).replace('{2,}', '*').getRegex()
+  text: edit(inline.gfm.text).replace('{2,}', '*').replace('{2,}', '*').getRegex()
```

Comment:
```suggestion
  text: edit(inline.gfm.text).replace(/\{2,\}/g, '*').getRegex()
```

you can use a regex to do a replace all

Reply:
Done.